### ISSUE TITLE
IndexHandler showing all routes

### ISSUE THREAD

**Issue by zwerch** [bahlo/goat#9]
I think it would be nice if `IndexHandler` would return all routes and endpoints, not only `GET` ones.
It would also be nice if they were sorted somehow (alphabetical by route maybe). I can't see an order here.

**Comment by bahlo** [comment]
I agree, let's show all routes with name and sort the index alphabetically. 

This will take place in the next minor version, but feel free to contribute if you want 👍
